pyproject.toml

```diff
@@ -22,7 +22,7 @@ dependencies = [
     'fpdf2',
     'scipy',
     'rich',
-    'harp-python'
+    'harp-python@git+<https://github.com/jsiegle/harp-python>#egg=decode-clock'
```

Comment:
we also need `spikeinterface[full]` for the drift map

Reply:
Thank you! I'll correct it.